src/index.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env browser */
+import React from 'react';
+
+import CoreApp from 'fusion-core';
+import {prepare} from 'fusion-react-async';
+import {html} from 'fusion-core';
+
+import {ApolloProvider} from 'react-apollo';
+
+import serverRender from './server';
+import clientRender from './client';
+
+import ProviderPlugin from './plugin';
+import ProvidedHOC from './hoc';
+import Provider from './provider';
+
+export default class App extends CoreApp {
+  constructor(root, getClient) {
+    super(root, el => {
+      return prepare(el).then(() => {
+        return __NODE__ ? serverRender(el) : clientRender(el);
```

Comment:
Might want to consider supporting passing through a render function here to support testing via enzyme

Reply:
I'll work on adding an example enzyme test, but I might make that a follow-up.